crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -95,24 +104,40 @@ where
         }
     }
 
-    fn default_election_config(num_nodes: u64) -> TYPES::ElectionConfigType {
-        StaticElectionConfig { num_nodes }
+    fn default_election_config(
+        num_nodes_with_stake: u64,
+        num_nodes_without_stake: u64,
+    ) -> TYPES::ElectionConfigType {
+        StaticElectionConfig {
+            num_nodes_with_stake,
+            num_nodes_without_stake,
+        }
     }
 
     fn create_election(
         entries: Vec<PeerConfig<PUBKEY>>,
         config: TYPES::ElectionConfigType,
     ) -> Self {
+        // get nodes with stake from `known_nodes_with_stake`
         let nodes_with_stake: Vec<PUBKEY::StakeTableEntry> = entries
             .iter()
             .map(|x| x.stake_table_entry.clone())
             .collect();
+        // make committee_nodes_with_stake to be nodes_with_stake
         let mut committee_nodes_with_stake: Vec<PUBKEY::StakeTableEntry> = nodes_with_stake.clone();
         debug!("Election Membership Size: {}", config.num_nodes);
-        committee_nodes_with_stake.truncate(config.num_nodes.try_into().unwrap());
+        // truncate committee_nodes_with_stake to only `num_nodes`
+        // since the `num_nodes_without_stake` are not part of the committee,
+        committee_nodes_with_stake.truncate(config.num_nodes_with_stake.try_into().unwrap());
```

Comment:
Is it safe to unwrap here?

Reply:
I think so, we were already doing this unwrap.  If we can't construct the election then we can't really run HotShot